src/main/java/Duke.java

```diff
@@ -1,10 +1,93 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static Boolean isFinished = false;
+    private static int itemCount = 0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner sc = new Scanner(System.in);
+        Task[] items = new Task[100];
+
+        printIntro();
+
+        while (!isFinished) {
+            String userInput = sc.nextLine();
+            String command = userInput.split(" ")[0];
+
+            switch (command) {
+            case "bye":
+                isFinished = true;
+                break;
+            case "list":
+                drawLine();
+                System.out.println("\tHere is your task list:");
+                for (int i = 0; i < itemCount; i++) {
+                    System.out.print("\t\t" + (i + 1) + ". ");
+                    System.out.println(items[i]);
+                }
+                drawLine();
+                break;
+            case "done": {
+                String arg = userInput.split(" ")[1];
+                int indexToMark = Integer.parseInt(arg) - 1;
+                items[indexToMark].markAsDone();
+                System.out.println("\tNice! I have marked this task as done:");
+                System.out.println("\t\t" + items[indexToMark]);
+                break;
+            }
+            case "todo": {
+                String arg = userInput.split(" ", 2)[1];
+                items[itemCount] = new Todo(arg);
+                incrementItemCount(items[itemCount]);
+                break;
+            }
+            case "deadline": {
+                String arg = userInput.split(" ", 2)[1];
```

Comment:
Perhaps you could refactor it into a separate method to with some comment to make it more readable?